test/support/application_generator.rb

```diff
@@ -64,7 +64,11 @@ def generate_files
         end
 
         rewrite_file(application.path("config/environments/test.rb")) do |c|
-          c.sub!(/config\.cache_classes\s*=\s*true/, "config.cache_classes = false")
+          if c.include?("config.cache_classes")
+            c.sub!(/config\.cache_classes\s*=\s*true/, "config.cache_classes = false")
+          else # 7.1+ doesn't have config.cache_classes in the config at all
+            c.sub!(/config.enable_reloading = false/, "config.enable_reloading = false\nconfig.cache_classes = false")
```

Comment:
Same as in acceptance_test.rb

```suggestion
            c.sub!(/config.enable_reloading = false/, "config.enable_reloading = true")
```